src/components/avatar.tsx

```diff
@@ -0,0 +1,21 @@
+import { twJoin } from "tailwind-merge";
+import user from "@/assets/icons/user.svg";
+
+interface AvatarProps {
+  imageUrl?: string;
+  className?: string;
+}
+
+export function Avatar(props: AvatarProps) {
+  const { imageUrl, className } = props;
+
+  console.log(imageUrl);
+
+  return (
+    <img
+      src={imageUrl === undefined ? user : imageUrl}
```

Comment:
Suggestion: create a variable and manage the value outside of the src property. Also, another suggestion is to change that "=== undefined" to "!= null", as in Javascript it will check for both null and undefined. (note that it is "!=" and NOT "!==")

Reply:
i just did that, thanks for the heads up